vnext/Microsoft.ReactNative/Views/DevMenu.cpp

```diff
@@ -113,30 +111,29 @@ void DevMenuManager::CreateAndShowUI() noexcept {
       Mso::React::ReactOptions::UseFastRefresh(m_context->Properties()) ? L"Disable Fast Refresh"
                                                                         : L"Enable Fast Refresh");
 
-  if (Mso::React::ReactOptions::GetJsiEngine(m_context->Properties()) == Mso::React::JSIEngine::Hermes) {
-      devMenu.SamplingProfilerText().Text(
-          !Microsoft::ReactNative::HermesSamplingProfiler::IsStarted() ? L"Start Hermes sampling profiler"
-                                                                       : L"Stop Hermes sampling profiler");
-      devMenu.SamplingProfilerIcon().Glyph(
-          !Microsoft::ReactNative::HermesSamplingProfiler::IsStarted() ? winrt::to_hstring("\uE1E5")
-                                                                       : winrt::to_hstring("\uE15B"));
-
-      std::ostringstream os;
-      if (Microsoft::ReactNative::HermesSamplingProfiler::IsStarted()) {
-        os << "Sampling profiler running..";
-      } else {
-        os << "Click to start.";
-      }
+  if (Mso::React::ReactOptions::JsiEngine(m_context->Properties()) == Mso::React::JSIEngine::Hermes) {
+    devMenu.SamplingProfilerText().Text(
+        !Microsoft::ReactNative::HermesSamplingProfiler::IsStarted() ? L"Start Hermes sampling profiler"
+                                                                     : L"Stop and copy trace path to clipboard");
+    devMenu.SamplingProfilerIcon().Glyph(
+        !Microsoft::ReactNative::HermesSamplingProfiler::IsStarted() ? L"\ue1e5" : L"\ue15b");
```

Comment:
fyi (we don't _need_ to do this but i figured i'd mention it anyway), it'd be better to set up a binding in the fonticon so that any time the IsStarted property changes it would automatically change the icon :)

Reply:
Thanks .. great to know that .. Being a novice in xaml,  It'll take me another day for me to figure out setting up the bindings :)